### PR TITLE
Code Jam Wrap Up

### PR DESCRIPTION
**Pull request by janine9vn**
This updates the website now that the code jam is wrapped up.

It corrects wording and sidebars to show that the code jam is no longer currently on going. This happens primarily on the main page, the events page, and the code jam page.

Additionally this PR adds some information about the Code Jam winners with links to their projects.

This PR also goes a bit out scope and edits the contents on the main Events page, namely removing Hacktoberfest, updating the 2021 events calendar, and re-organizing the existing event blocks in an order that makes more sense.

### CODE REVIEW

File: pydis_site/templates/events/index.html [hunk 1]
```diff
@@ -9,58 +9,34 @@
...
+        <p>Each year, we organize at least one code jam, one during the summer and sometimes one during the winter. During these events, members of our community will work together in teams to create something amazing using a technology we picked for them. One such technology that was picked for the Summer 2021 Code Jam was text user interfaces (TUIS), where teams could pick from a pre-approved list of frameworks.</p>
+        <p>To help fuel the creative process, we provide a specific theme, like <strong>Think Outside the Box</strong> or <strong>Early Internet</strong>. At the end of the Code Jam, the projects are judged by Python Discord server staff members and guest judges from the larger Python community. The judges will consider creativity, code quality, teamwork, and adherence to the theme.</p>
```
Comment (Bluenix2) [inline pydis_site/templates/events/index.html]: Why didn't these get cut off like the PyWeek ones?

File: pydis_site/templates/events/sidebar/code-jams/previous-code-jams.html [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <div class="box">
     <p class="menu-label">Previous Code Jams</p>
     <ul class="menu-list">
+        <li><a class="has-text-link" href="{% url "events:page" path="code-jams/8" %}">Code Jam 8: Think Outside the Box</a></li>
```
Comment (Bluenix2) [inline pydis_site/templates/events/sidebar/code-jams/previous-code-jams.html]: The theme was actually "Think Inside the Box".

File: pydis_site/templates/events/index.html (hunk 1 above)
Comment (janine9vn) [reply pydis_site/templates/events/index.html]: It's not cut off, it just doesn't have an icon to show, so the text takes the full width.
But this did make me realize the Advent of Code icon wasn't showing up, so I'm now fixed that.

Comment (Bluenix2) [reply pydis_site/templates/events/index.html]: No I mean that PyWeek's lines you cut off at a certain length, while these ones go on for several hundreds of characters.

Comment (janine9vn) [reply pydis_site/templates/events/index.html]: I'm not sure what you're referencing. The length of the line is entirely dependent on if the word can fit or if it needs to go on the next line. Do you have a screenshot?

Comment (Bluenix2) [reply pydis_site/templates/events/index.html]: Why doesn't these lines look like this?
image

Comment (janine9vn) [reply pydis_site/templates/events/index.html]: Oh, you mean within the file itself, not how it displays. 

That's just what it was from the start. I only edited the contents of that page and the general order, not how the lines were broken up. I also tend to have word wrap turned on in my editor so it doesn't really bother me.

File: pydis_site/templates/events/index.html [hunk 3]
```diff
@@ -9,58 +9,34 @@
...
-                </p>
-                <p>
-                    The participants will have ten days to create a game using the technology we've selected, and drawing
-                    inspiration from a provided theme. After the event, a panel of judges will play all the games and select a
-                    winner. The top 5 will featured in a special video on our <a href="https://www.youtube.com/channel/UCQsrA4xo6jvdgsJZhKaBL6w">YouTube channel</a>.
-                </p>
-                <p>
-                    The <a class="has-text-link" href="{% url "events:page" path="game-jams/2020" %}">first edition of the Game Jam</a> ran from
-                    <strong>April 17, 2020 to April 26, 2020</strong>.
+                    For the past 15 years, <a href="https://pyweek.org">PyWeek</a> has been running a bi-annual game jam for the
```
Comment (fisher60) [inline pydis_site/templates/events/index.html]: This href should probably have `target="_blank"` and `rel="noopener"` the former for UX, the latter for security. Likewise with any other external links that may disrupt reading content on our site when the user may want to continue.

File: pydis_site/templates/events/pages/code-jams/8/_index.html [hunk 4]
```diff
@@ -20,10 +20,54 @@
...
+        <h4><i class="fa fa-medal"></i> Lovable Lobsters: Ultimate Tic Tac Toe</h4>
+        <p><em>A5Rocks, Bast, Dacheat, mega_hirtz, CopOnTheRun, richphi</em></p>
+        <p>
+            Thinking inside a box, that is inside a box, that is inside yet another box.
+
+            The terminal program created by the Lovable Lobsters allows you to play Ultimate Tic Tac Toe right form your terminal. The really impressive part though? You can play with your friends and family over your network! Their program has a server-client set-up that lets you play with your friends and family from different computers.
+            <br/>
+            <a href="https://www.youtube.com/watch?v=WI9tgQeAfXw" title="Lovable Lobsters Demo Video" target="_blank" rel="noopener"><i class="fa fa-video"> </i> Demo video</a>
+            <br/>
+            <a href="https://github.com/A5rocks/code-jam-8" title="Lovable Lobsters GitHub Repository" target="_blank" rel="noopener"><i class="fa fa-github"></i> Githib Repository</a>
```
Comment (kosayoda) [inline pydis_site/templates/events/pages/code-jams/8/_index.html]:
```suggestion
            <a href="https://github.com/A5rocks/code-jam-8" title="Lovable Lobsters GitHub Repository" target="_blank" rel="noopener"><i class="fa fa-github"></i> Github Repository</a>
```
Multiple instances of this throughout the page as far as I can tell~

File: pydis_site/templates/events/pages/code-jams/8/_index.html [hunk 5]
```diff
@@ -20,10 +20,54 @@
...
+            <br/>
+        </p>
+
+        <h4><i class="fa fa-medal"></i> Robust Reindeer: Rubick's Cube</h4>
```
Comment (kosayoda) [inline pydis_site/templates/events/pages/code-jams/8/_index.html]:
```suggestion
        <h4><i class="fa fa-medal"></i> Robust Reindeer: Rubik's Cube</h4>
```

**Review by fisher60 — APPROVED** [review]
All looks good to me.

File: pydis_site/templates/events/pages/code-jams/8/_index.html [hunk 6]
```diff
@@ -36,14 +80,14 @@ <h3 id="technology"><a href="#technology">Technology</a></h3>
     <p>
         The chosen technology/tech stack for this year is <strong>Text User Interfaces</strong> (TUIs).
         Each team must create a program with one of <a href="{% url "events:page" path="code-jams/8/frameworks" %}">the approved frameworks</a> that creates a user interface that is text based.
-        For more information of TUIs and what's involved with such an interface, check out <a href="https://en.wikipedia.org/wiki/Text-based_user_interface">this wikipedia article</a>.
+        For more information of TUIs and what's involved with such an interface, check out <a href="https://en.wikipedia.org/wiki/Text-based_user_interface" target="_blank" rel="noopener">this wikipedia article</a>.
     </p>
     <h3 if="qualifier"><a href="#qualifier">The Qualifier</a></h3>
```
Comment (kosayoda) [inline pydis_site/templates/events/pages/code-jams/8/_index.html]:
```suggestion
    <h3 id="qualifier"><a href="#qualifier">The Qualifier</a></h3>
```

**Review by kosayoda — CHANGES_REQUESTED** [review]
Good work. I think the `Code Jam Winners` section could use a lot more whitespace, but that isn't a blocker for this PR.

File: pydis_site/templates/events/pages/code-jams/8/_index.html (hunk 4 above)
Comment (janine9vn) [reply pydis_site/templates/events/pages/code-jams/8/_index.html]: ;-; ... You know, you'd think I'd be able to spell GitHub correctly at this point.

**Review by wookie184 — APPROVED** [review]
Good stuff

File: pydis_site/templates/events/sidebar/events-list.html [hunk 7]
```diff
@@ -1,10 +1,10 @@
 <div class="box">
```
Comment (Bluenix2) [inline pydis_site/templates/events/sidebar/events-list.html:1]: What is this by the way?
image

It is on the production site as well: https://pythondiscord.com/events/

File: pydis_site/templates/events/pages/code-jams/8/_index.html [hunk 8]
```diff
@@ -20,10 +20,58 @@
...
+        <h4 class="mt-5 mb-2"><i class="fa fa-medal"></i> Robust Reindeer: Rubik's Cube</h4>
+        <p class="my-1"><em>Björn, aaronshenhao, mathsman, Dude Saber, 詭異, Keppo</em></p>
+        <p class="my-1">
+            This submission is a Rubik's cube, rendered in a text user interface (that was a constraint) using the asciimatics package, and addressing the theme "thinking inside the box".
+
+            Just like a real world Rubik's cube, you can move this cube around to look at it from all sides. And, of course, you can rotate the individual discs it is made up of to first scramble up the order and then to try and solve it into perfect ordering again.
+        </p>
+        <p>
+            <a href="https://github.com/bjoseru/pdcj8-robust-reindeer" title="Robust Reindeer GitHub Repository" target="_blank" rel="noopener"><i class="fa fa-github"></i> GitHub Repository</a>
+            <br/>
```
Comment (Bluenix2) [inline pydis_site/templates/events/pages/code-jams/8/_index.html:62]: Where's the video for this team?

File: pydis_site/templates/events/sidebar/events-list.html (hunk 7 above)
Comment (whispersofthedawn) [reply pydis_site/templates/events/sidebar/events-list.html:1]: It's the [breadcrumb](https://bulma.io/documentation/components/breadcrumb/) for the page.

File: pydis_site/templates/events/pages/code-jams/8/_index.html (hunk 8 above)
Comment (janine9vn) [reply pydis_site/templates/events/pages/code-jams/8/_index.html:62]: Purposely not included.

**Review by Bluenix2 — APPROVED** [review]
Now that my two questions have been resolved, looks good!

**Pull request merged**